### PR TITLE
.gitignore and CMake

### PR DESCRIPTION
**Pull request by chkwon**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,19 @@
-*.o
+
+# Compiled executable and output files
 genvrp
```
Comment (vidalt) [inline .gitignore:3]: If genvrp is created in the build directory, then it does not need a specific rule anymore

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+add_executable(bin
+        Program/main.cpp
+        ${src_files}
+        )
+set_target_properties(bin PROPERTIES OUTPUT_NAME genvrp)
+
+# Test Executable
+enable_testing()
+add_test(NAME    test_bin
+         COMMAND genvrp ../Instances/CVRP/X-n101-k25.vrp mySolution.sol -seed 1)
```
Comment (vidalt) [inline CMakeLists.txt:27]: Perhaps naming the executable ./hgs could be more transparent.
./genvrp is a legacy name that does not represent much anymore (not a real terminology in the paper/project)

**Comment by vidalt** [comment]
- I noticed a few legacy references in the README.md that refer to the old `make test` syntax. It would be great to update these sentences with the correct line of command when using CMake.
- Should we keep the old Makefile? The good side is that it permits to keep the same compilation workflow for whoever used it so far. The bad part is that it's clearly depreciated if we adopt CMake as a standard for this project.

File: CMakeLists.txt (hunk 2 above)
Comment (chkwon) [reply CMakeLists.txt:27]: Renaming it to ./hgs makes sense.

File: .gitignore (hunk 1 above)
Comment (chkwon) [reply .gitignore:3]: Sure, I'll remove it :)

**Comment by chkwon** [comment]
* I will replace `make test` with `cmake`. 
* It seems you are using `Makefile` for CI Actions now. Let's just leave `Makefile` there for now.  When the library version is available, I think we can write some better testing codes and better CI actions based on those.

**Comment by chkwon** [comment]
When some additional changes are required in a PR, you can just leave it without merging it until all issues are resolved.  If I make some new changes to resolve such issues, this PR will be automatically updated. 

Once you merged the PR, we need to create a new PR (like #9).  Just something to consider in the future :)

**Comment by vidalt** [comment]
Thank for the tip!

**Pull request merged**